apps/admin-page-e2e/src/integration/pages/account.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { IUser } from '@interfaces';
+import { LocalStorageVars, UserRoles } from '@models';
+import { AuthenticationService, AuthUserEnum } from '@webstore/mocks';
+
+const mockAdminUser: IUser = {
+  userId: 'PleaseTakeMe',
+  email: 'e2e@test.com',
+  roles: [UserRoles.user, UserRoles.admin],
+  name: 'telinior the great',
+  phoneNumber: '69420420',
+  streetAddress: 'Y u b like dis?',
+  streetAddress2: 'plds stap',
+  city: 'TestCity',
+  postcode: '6996',
+  country: 'Neverland',
+};
+
+const authMockService = new AuthenticationService();
+
+describe('Account page for admin user', () => {
```

Comment:
Missing tests (and functionality) for:
- Update Info button should be disabled if the information hasn't been changed or is invalid
- The update info button should have a tooltip explaining why it is disabled (when the button is disabled)
- Tests for navigating to the page by clicking on the navbar
- Tests for navigating to the dashboard via the sidebar (since you added the routerlink for it)
- Tests for failure to load user information when accessing the page
- Tests for the failure of updating the user information
- Tests for the failure of updating the password